output/documentation/how-to-release/index.html

```diff
@@ -210,7 +210,7 @@ <h4 id="5-send-a-vote-e-mail-to-dev-parquet-apache-org">5. Send a VOTE e-mail to
 * https://dist.apache.org/repos/dist/dev/parquet/&lt;PATH&gt;
```

Comment:
I just want to confirm that the link 'https://dist.apache.org/repos/dist/' is not going to be deprecated. Correct?

Reply:
Correct.
> The standard way of uploading releases[1] will STILL apply (...)
> [1] https://www.apache.org/legal/release-policy.html#upload-ci

Reply:
LGTM